### PR TITLE
[docs] SMTP config

### PR DESCRIPTION
**Pull request by Anddraca**
## What changes are proposed in this pull request?

Added docs and associated images for configuring SMTP connections

## Summary by CodeRabbit

- **New Features**
	- Introduced a "New User Invitations" section for onboarding users via invitation links.
	- Added functionality for automatic email invitations through SMTP configuration.
  
- **Documentation**
	- Updated documentation to clarify authentication modes and user management processes.

### CODE REVIEW

File: docs/source/teams/pluggable_auth.md [hunk 1]
```diff
@@ -72,6 +70,56 @@ screen, and provide the `FIFTYONE_AUTH_SECRET` to login.
 
 (identity-providers)=
 
+### New User Invitations
+
+As of FiftyOne Teams 2.1.0, onboarding new users can be done via invitation
```
Comment (swheaton) [inline docs/source/teams/pluggable_auth.md:75]: 2.2

File: docs/source/teams/pluggable_auth.md [hunk 2]
```diff
@@ -72,6 +70,56 @@ screen, and provide the `FIFTYONE_AUTH_SECRET` to login.
...
+links. To do so, "Enable invitation" must be toggled on in the Organizations
+section of the [Super Admin UI](#super-admin-ui).
+
+![Enable Invitations](/images/teams/cas/org_enable_invitations.png)
+
+This allows creating invitation links in [internal mode](#internal-mode) that
+can be manually sent to users. When those users click the links, they will be
+added to the Organization and prompted to log in.
+
+As of FiftyOne Teams 2.2.0, these invitation links can be automatically sent
```
Comment (swheaton) [inline docs/source/teams/pluggable_auth.md:85]: New section

**Review by swheaton — APPROVED** [review]
nice! I also did not know we could use markdown, I thought RST was required 🤔 
Docs looked good to me. Product will probably want to proof at some point.

**Review by brimoor — APPROVED** [review]
LGTM

**Pull request merged**